Re-add contact page and add hard-coded JSON data

nmsulliv opened the pull request:
The contact page I added last Friday somehow got deleted locally so I added it again. I also added three hard coded messages to my homepage.

tc466 commented:
> The contact page I added last Friday somehow got deleted locally so I added it again. I also added three hard coded messages to my homepage.

It looks like the reason the contact page is missing is because the `NicoleDev` branch is synced to an older commit (when the contact page has not been added yet) rather than the most recent master branch (illustrated below). Creating another contact page would create a conflict when the pull request is merged with the master branch (because now there are two versions of the page).

```
          A---B---C NicoleDev
         /
    D---E---F---G master
```

It is completely fine that the contact page does not exist in the `NicoleDev` branch. When the pull request is closed, the `NicoleDev` will get merged with master, and the merge result will contain both the contact page as well as the newly added JSON feature.

nmsulliv commented:
Thank you, I undid the addition of the extra contact page in my latest commit.

nmsulliv commented:
> > The contact page I added last Friday somehow got deleted locally so I added it again. I also added three hard coded messages to my homepage.
> 
> It looks like the reason the contact page is missing is because the `NicoleDev` branch is synced to an older commit (when the contact page has not been added yet) rather than the most recent master branch (illustrated below). Creating another contact page would create a conflict when the pull request is merged with the master branch (because now there are two versions of the page).
> 
> ```
>           A---B---C NicoleDev
>          /
>     D---E---F---G master
> ```
> 
> It is completely fine that the contact page does not exist in the `NicoleDev` branch. When the pull request is closed, the `NicoleDev` will get merged with master, and the merge result will contain both the contact page as well as the newly added JSON feature.

Thank you, I undid the addition of the extra contact page in my latest commit.
